CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+* Added the use of sync.Pool in EncoderMap and renamed it to MultiEncoder
```

Comment:
Renaming in internal packages no need to be logger into public changelog. Most important: supported pool of encoders, which implement  ResetableWriter interface